components/user-popover.js

```diff
@@ -0,0 +1,75 @@
+import { USER_FULL } from '@/fragments/users'
+import Moon from '@/svgs/moon-fill.svg'
+import { useQuery } from '@apollo/client'
+import Link from 'next/link'
+import { useEffect, useRef, useState } from 'react'
+import { Popover } from 'react-bootstrap'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import { User } from './user-list'
+
+function StackingSince ({ since }) {
+  return (
+    <small className='text-muted d-flex-inline'>stacking since: {since
+      ? <Link href={`/items/${since}`}>#{since}</Link>
+      : <span>never</span>}
+    </small>
+  )
+}
+
+export default function UserPopover ({ text, children }) {
+  const [user, setUser] = useState(null)
+  const [isTriggered, setIsTriggered] = useState(false)
+  const { data } = useQuery(USER_FULL, {
+    variables: { name: text.replace('@', '') },
```

Comment:
I think passing `name` would make more sense if that's all you need from text, see my other comment

Reply:
See comment above